qa/suites/fs/thrash/workloads/overrides/prefetch_dirfrags

```diff
@@ -0,0 +1 @@
+.qa/cephfs/overrides/prefetch_dirfrags
```

Comment:
There already has a directory `prefetch_dirfrags`, so you are adding a new `prefetch_dirfrags` file here.

Reply:
Thanks